src/qt/modaloverlay.cpp

```diff
@@ -169,4 +172,4 @@ void ModalOverlay::closeClicked()
 {
     showHide(true);
     userClosed = true;
-}
+}
```

Comment:
Sorry there was a bad commit-split in #11237 which meant the EOF newline got deleted and readded in the next commit, might want to squash them or just fix this